src/menus/share-panel.ts

```diff
@@ -66,55 +67,74 @@ export class IauxSharingOptions extends LitElement {
   }
 
   loadProviders() {
-    const { baseHost, creator, description, identifier, type, fileSubPrefix } =
-      this;
-    const params = {
-      baseHost,
-      creator,
-      description,
-      identifier,
-      type,
-      fileSubPrefix,
-    } as unknown as ProviderParams;
+    let shareUrl = `https://${this.baseHost}/details/${this.identifier}`;
+    if (this.fileSubPrefix) {
+      shareUrl += `/${this.fileSubPrefix}`;
+    }
+    const shareBlurb = [
+      this.description,
+      this.creator,
+      'Free Download, Borrow, and Streaming',
+      'Internet Archive',
+    ]
+      .filter(Boolean)
```

Comment:
q: why do we need to filter? should we have defaults for the desc, creator?

Reply:
Sometimes things didn't have a creator was the main case.